### ISSUE TITLE
Alert error state: icon screen reader output

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2943]
## Problem
Today our errors alerts use an icon to convey visually that there is an error there must be a textual indication of some kind but this same indication is not available via screen reader. If a screen reader user simply reads the text on the screen right now all they hear for example is ‘Enter the name of your organization’, without the context of alternative text or the word Error there is no way for them to know that this is an error, which is not an equivalent experience to sighted users as they are able to see the icon.

![Image](https://github.com/user-attachments/assets/bfd94c1f-4273-4b40-977f-b43f683da107)

## Acceptance criteria
- [ ] For **all** of the alerts in the registrar the error alert icon has the screen reader output of `Error`

### Prioritization Comments
**WCAG Conformance Level:**  2.1 AA
**WCAG Standard that applies:** 
3.3.1 - Error Identification (Critical)
1.3.1 - Info and Relationships (Critical)
4.1.2 - Name, Role, Value (Critical)
**Critical Failure if Not Met?:**  Yes
**Prioritization:** Blocker
**Reason:** This issue critically impacts accessibility by preventing screen reader users from identifying error alerts, resulting in a non-equivalent experience compared to sighted users. Providing a screen reader-readable indication of errors is essential to ensure all users can understand when an input error has occurred. This can be achieved by implementing a screen reader output of "Error" for the alert icon.
